### ISSUE TITLE
Renaming repository

### ISSUE THREAD

**Issue by jcarpent** [Gepetto/gepetto-viewer-corba#108]
It appears that the naming of this repository is no more the correct one and should be replaced by **gepetto-gui**. @jmirabel @nim65s What do you think?

**Comment by nim65s** [comment]
I would be in favor of a shorter name, but apart from that I don't mind. I can handle the rename if necessary.

**Comment by jcarpent** [comment]
**gepetto-gui** looks good.

**Comment by jmirabel** [comment]
That will conflict with future changes currently only on my fork.
- I moved the CORBA server into a non-mandatory plugin.
- I moved all the code that is independent to CORBA into gepetto-viewer.
- To come some day: possibility to interact with OSG widgets trough the Python interpreter without CORBA.

I was even thinking of moving the plugin into a subdirectory of gepetto-viewer and make it optional. Then, I would remove gepetto-viewer-corba.

Given those changes, the package to rename is gepetto-viewer and not gepetto-viewer-corba.
